src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -285,280 +306,109 @@
      */
     public static final Supplier<Key<Value<ArtType>>> ART = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "ART");
 
     /**
-     * Represents the {@link Key} for the attachment {@link Surface}
-     * of a button or lever.
+     * The attachment {@link Surface} of a button or lever {@link BlockState}
      */
     public static final Supplier<Key<Value<Surface>>> ATTACHMENT_SURFACE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "ATTACHMENT_SURFACE");
 
     /**
-     * Represents the {@link Key} for the damage dealt by a
-     * {@link DamagingProjectile}, e.g. an {@link ArrowEntity}.
+     * The damage dealt by an {@link ArrowEntity} on impact.
      */
     public static final Supplier<Key<BoundedValue<Double>>> ATTACK_DAMAGE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "ATTACK_DAMAGE");
 
     /**
-     * Represents the {@link Key} for the time of a {@link Ravager} is considered attacking.
+     * The time of a {@link Ravager} is considered attacking.
      */
     public static final Supplier<Key<Value<Integer>>> ATTACK_TIME = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "ATTACK_TIME");
 
     /**
-     * Represents the {@link Key} for representing the {@link Axis} direction
-     * of a {@link BlockState}.
+     * The author of a {@link ItemTypes#WRITTEN_BOOK} {@link ItemStack}.
+     */
+    public static final Supplier<Key<Value<Text>>> AUTHOR = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "AUTHOR");
+
+    /**
+     * The {@link Axis} direction of a {@link BlockState}.
      */
     public static final Supplier<Key<Value<Axis>>> AXIS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "AXIS");
 
     /**
-     * Represents the {@link Key} for a {@link Banner}'s base {@link DyeColor}.
+     * The ticks until a {@link Ageable} turns into an adult.
      */
-    public static final Supplier<Key<Value<DyeColor>>> BANNER_BASE_COLOR = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BANNER_BASE_COLOR");
+    public static final Supplier<Key<BoundedValue<Integer>>> BABY_TICKS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BABY_TICKS");
 
     /**
-     * Represents the {@link Key} for a {@link Banner}'s patterns.
+     * The {@link BannerPatternLayer}s of a {@link Banner}.
      */
     public static final Supplier<Key<ListValue<BannerPatternLayer>>> BANNER_PATTERN_LAYERS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BANNER_PATTERN_LAYERS");
 
     /**
-     * Represents the {@link Key} for the width of the physical form of an
-     * {@link Entity}.
+     * The base {@link DyeColor} of a {@link Banner} or {@link TropicalFish}.
+     */
+    public static final Supplier<Key<Value<DyeColor>>> BASE_COLOR = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BASE_COLOR");
```

Comment:
Is it worth merging this into `DYE_COLOR`?

Reply:
done it